### ISSUE TITLE
Turkish translations crash in the MetricsView for 1.7.1

### ISSUE THREAD

**Issue by crankycoder** [mozilla/MozStumbler#1514]
@asteko - I'm getting crashes in the MetricsView from turkish users.

I'm backing out the turkish translation until we can have that resolved.

``` java
Stack Trace     
java.lang.RuntimeException: Unable to resume activity {org.mozilla.mozstumbler/org.mozilla.mozstumbler.client.navdrawer.MainDrawerActivity}: java.util.UnknownFormatConversionException: Conversion: p
at android.app.ActivityThread.performResumeActivity(ActivityThread.java:2621)
at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:2649)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2122)
at android.app.ActivityThread.access$700(ActivityThread.java:134)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1218)
at android.os.Handler.dispatchMessage(Handler.java:99)
at android.os.Looper.loop(Looper.java:137)
at android.app.ActivityThread.main(ActivityThread.java:4867)
at java.lang.reflect.Method.invokeNative(Native Method)
at java.lang.reflect.Method.invoke(Method.java:511)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1007)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:774)
at dalvik.system.NativeStart.main(Native Method)
Caused by: java.util.UnknownFormatConversionException: Conversion: p
at java.util.Formatter$FormatToken.unknownFormatConversionException(Formatter.java:1397)
at java.util.Formatter$FormatToken.checkFlags(Formatter.java:1334)
at java.util.Formatter.transform(Formatter.java:1440)
at java.util.Formatter.doFormat(Formatter.java:1079)
at java.util.Formatter.format(Formatter.java:1040)
at java.util.Formatter.format(Formatter.java:1009)
at java.lang.String.format(String.java:1998)
at java.lang.String.format(String.java:1972)
at org.mozilla.mozstumbler.client.navdrawer.MetricsView.updatePowerSavingsLabels(MetricsView.java:161)
at org.mozilla.mozstumbler.client.navdrawer.MetricsView.update(MetricsView.java:219)
at org.mozilla.mozstumbler.client.navdrawer.MainDrawerActivity.onPostResume(MainDrawerActivity.java:203)
at android.app.Activity.performResume(Activity.java:5134)
at android.app.ActivityThread.performResumeActivity(ActivityThread.java:2611)
```

**Comment by asteko** [comment]
I was just about to file an issue that the app v1.7.1 cannot be launched with Turkish interface. Changing phone's language to English helps, and the app can be lauched with English interface.

Tried clean install too, but it doesn't help. Using Samsung Galaxy S3 Mini on Android 4.1.2

**Comment by crankycoder** [comment]
Thanks.  @garvankeeley just mentioned that he thinks he can get this fixed.

**Comment by crankycoder** [comment]
Fixed in #1517

**Comment by asteko** [comment]
Hotfix v1.7.2 works without any problem with Turkish interface. Thanks.
